crates/virtio-queue/Cargo.toml

```diff
@@ -13,13 +13,13 @@ edition = "2018"
 test-utils = []
 
 [dependencies]
-vm-memory = "0.7.0"
+vm-memory = ">=0.7.0"
```

Comment:
Should we also update to >=0.8.0 if it's the newest release at this point?

Reply:
Great question. I intentionally didn't do that since that version is not strictly required for the functionality of this crate to work.